ftw/footer/tests/test_footer.py

```diff
@@ -1,5 +1,10 @@
+from unittest2 import skipIf
```

Comment:
In most packages we sort imports case sensitive I think.

Reply:
it seems not in ftw.builder's `test_dexterity.py` (or further down here).   I don't mind tho